lxqthtmldelegate.cpp

```diff
@@ -98,8 +98,8 @@ void HtmlDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
 
     // set text colors
     QAbstractTextDocumentLayout::PaintContext ctx;
-    QPalette::ColorGroup colorGroup = (option.state & QStyle::State_Active) ? QPalette::Active : QPalette::Inactive;
-    if (option.state & QStyle::State_Selected) // selected items
+    QPalette::ColorGroup colorGroup = (option.state & QStyle::State_Active) != 0u ? QPalette::Active : QPalette::Inactive;
+    if ((option.state & QStyle::State_Selected) != 0u) // selected items
```

Comment:
I really doubt that substitutions like the above ones could make the code more readable (there are some in other places too). I'm afraid I wanted to say the same thing about too much usage of `nullptr` but `0u` might feel strange to code readers — especially if they're contributors who want to preserve the code style.

That being said, if you prefer them, there will be no objection from me.

Reply:
Will come back to you on this.